src/App.tsx

```diff
@@ -21,6 +32,10 @@ export const App: React.FC = () => {
             data-cy="perPageSelector"
             id="perPageSelector"
             className="form-control"
+            value={perPage}
+            onChange={event => {
+              handleValueChange(+event.target.value);
+            }}
           >
             <option value="3">3</option>
             <option value="5">5</option>
```

Comment:
consider to create array and map options 
```
const options = [3, 5, 10, 20];
```

```

```suggestion
  {options.map(value => <option key={value} value={value}>{value}</option>)}
```